### PR TITLE
Strip email

### PR DESCRIPTION
**Pull request by batshoes**

### CODE REVIEW

**Comment by batshoes** [comment]
Strip incoming email of any html tag and then test against regex.

File: app/controllers/tickets_controller.rb [hunk 1]
```diff
@@ -28,6 +29,16 @@ def set_client
     @session = params[:session]
   end
 
+  def check_email(email)
+    regex = /\S+@\S+[\.][0-9a-z]+/
+    stripped = strip_tags(email)
+    test = regex.match stripped
```
Comment (paniagua) [inline app/controllers/tickets_controller.rb:35]: `test` what? the variable name does not express anything meaningful

File: app/controllers/tickets_controller.rb [hunk 2]
```diff
@@ -28,6 +29,16 @@ def set_client
...
+    unless test
+      response.status = 401
```
Comment (paniagua) [inline app/controllers/tickets_controller.rb:37]: does it mean that if `test` fails then the request is not `authorized`? just checking if it is the right response

File: app/controllers/tickets_controller.rb [hunk 3]
```diff
@@ -42,7 +53,9 @@ def set_params
     ticket_attributes = {}
     ticket_number = 1
     @tickets.each do |t|
-      ticket_attributes["ZenDeskTicket#{ticket_number}"] = t["zendesk_url"]
+      
+      ticket_attributes["Ticket: #{ticket_number}"] = t["status"] + ": " + t["zendesk_url"]
```
Comment (paniagua) [inline app/controllers/tickets_controller.rb:57]: `ticket_attributes["Ticket: #{ticket_number}"] = t["status"] + ": " + t["zendesk_url"]` =>
``ticket_attributes["Ticket: #{ticket_number}"] = "#{t['status']}: #{t['zendesk_url']}"`

**Pull request merged**